lxc/Deploy.sh

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
+
+# Initially need to config LXD on host machine. Run:
+# sudo lxd init
+# Init config should be like:
+#
+# config:
+#   images.auto_update_interval: "0"
+# networks: []
+# storage_pools:
+# - config:
+#     size: 30GiB
+#   description: ""
+#   name: default
+#   driver: zfs
```

Comment:
Is `zfs` a default storage driver for LXC/Jepsen? Does it need additional kernel modules to be installed? AFAIK, it's not shipped with vanilla Linux kernel.

Reply:
Made config as example, not mandatory. What storage you can recommand?

Reply:
Something available by default in the most popular distributions (like Ubuntu)